ts/lib/shortcuts.ts

```diff
@@ -0,0 +1,149 @@
+// Copyright: Ankitects Pty Ltd and contributors
+// License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+import * as tr from "./i18n";
+
+export type Modifier = "Control" | "Alt" | "Shift" | "Meta";
+
+const modifiers: Modifier[] = ["Control", "Alt", "Shift", "Meta"];
+
+// how modifiers are mapped
+const platformModifiers =
+    navigator.platform === "MacIntel"
+        ? ["Meta", "Alt", "Shift", "Control"]
+        : ["Control", "Alt", "Shift", "OS"];
+
+function splitKeyCombinationString(keyCombinationString: string): string[][] {
+    return keyCombinationString.split(", ").map((segment) => segment.split("+"));
+}
+
+function modifiersToPlatformString(modifiers: string[]): string {
+    const displayModifiers =
+        navigator.platform === "MacIntel"
```

Comment:
Maybe we can split these checks into a separate function, or move isApplePlatform() https://github.com/ankitects/anki/blob/a20606530ed181b75046c3c4d2b2bae06382c022/ts/sveltelib/shortcuts.ts#L4 into this module?

Reply:
Honestly, I wasn't aware of this file. May I ask you why you put it into sveltelib? When I created it, I had in mind functionality that depended on Svelte in some way (e.g. Svelte stores). For that reason I'd be for moving `isApplePlatform`. Just wonder what your reasoning was for putting them under sveltelib.

Reply:
I only added it yesterday, so not surprised you missed it :-) You're right that it probably belongs in lib/ instead; my thinking at the time was just that it's more GUI related than logic-related, but lib/ already contains other GUI stuff.